devops/deploy/spin-up-script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Just a basic script to spin up listening servers on different ports
+PORTS=$(seq 30000 30050)
```

Comment:
Please use the dynamic range specified in [RFC6335 S.6 Port Number Ranges](https://www.rfc-editor.org/rfc/rfc6335.html#section-6): 49152 - 65535

See also: [Service Name and Transport Protocol Port Number Registry](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml)

Reply:
done!